packages/artifact/__tests__/upload.test.ts

```diff
@@ -362,7 +362,9 @@ describe('Upload Tests', () => {
     const uploadHttpClient = new UploadHttpClient()
     expect(
       uploadHttpClient.patchArtifactSize(-2, 'my-artifact')
-    ).rejects.toThrow('Unable to finish uploading artifact my-artifact')
+    ).rejects.toThrow(
+      'Patch Artifact Size failed: Artifact service responded with 400'
```

Comment:
Same comment around the verb `Patch`, which is a REST implementation detail, we probably should shy away from details, but describe the symptom?

Reply:
Removed `Patch Artifact Size` and changed it to `Finalize artifact upload`